README.md

```diff
@@ -1,31 +1,42 @@
-Welcome to the GC Talent Monorepo.
+# Welcome to the GC Digital Talent monorepo
 
-The GC Talent app is divided into multiple services, each treated as its own subproject:
-- /api, the API service
-- /admin, a Client app for an admin dashboard
-- /auth, an OIDC authentication service
+The GC Digital Talent app is divided into multiple services, each treated as its own sub-project:
+- `/api`, the API service
+- `/admin`, a Client app for an admin dashboard
+- `/auth`, an OpenID Connect (OIDC) authentication service
 
-Each service is meant to run in a seperate container. However, since they all use the Laravel or Lumen framework, they can also be run on a single php server, with requests routed carefully between them. (This is how the docker-compose setup in /infrastructure works.)
+Each service is meant to run in a separate container. However, since they all use the [Laravel](https://github.com/laravel/laravel) or [Lumen](https://github.com/laravel/lumen) framework, they can also be run on a single PHP server, with requests routed carefully between them (this is how the docker-compose setup in `/infrastructure` works).
 
-Each subproject has its own README, with advice on getting it configured. If you want to run the project locally using Docker, check out /infrastructure/README.md first.
+Each sub-project has its own `README.md`, with advice on setup and configuration. If you want to run the project locally using Docker, check out `/infrastructure/README.md` first.
+
+Finally, head on into the relevant directory for the service you'd like to work on and follow the steps in its respective `README.md` file to get the service running locally.
 
 ## Dependencies
+### Development Setup
+
+You'll need to install:
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+- [Node & NPM](https://nodejs.org/en/)
+- [PHP 7.4](https://www.php.net/downloads)
+- [Composer](https://getcomposer.org/)
```

Comment:
Technically you can actually avoid installing these by using Docker.

Reply:
Indeed, this ties in a bit to the discussion about Docker-only instructions or not.